src/cts/src/TritonCTS.i

```diff
@@ -141,14 +144,14 @@ void
 set_distance_between_buffers(double distance)
 {
   getTritonCts()->getParms()->setSimpleSegmentsEnabled(true);
-  getTritonCts()->getParms()->setBufferDistance(distance);
+  getTritonCts()->getParms()->setBufferDistance(distance * getDb()->getChip()->getBlock()->getDbUnitsPerMicron());
```

Comment:
When converting from um to DBU I would recommend using `std::round()` because the implicit cast from float to int will truncate.

Reply:
Done.